docs/src/index.tsx

```diff
@@ -112,10 +107,7 @@ const App = () => {
 
     const queryStream = await streamQueryV2({
       streamQueryConfig,
-      onStreamEvent,
-      onError: (error) => {
-        console.error("Error", error);
-      },
+      onStreamEvent
```

Comment:
Type issue: We no longer support an `onError`. We should IMO and I'll add an issue for it.

Reply:
Errors are now emitted as part of the stream IIRC.